tests/test_basic.py

```diff
@@ -1528,6 +1531,39 @@ def test_lbs_matrix(self):
         self.assertTrue(M.shape[0] == V.shape[0])
         self.assertTrue(M.shape[1] == W.shape[1]*4)
 
+    def test_direct_delta_mush(self):
+        V, F = igl.read_triangle_mesh(os.path.join(self.test_path, "arm.obj"))
+        W = igl.read_dmat(os.path.join(self.test_path, "arm-weights.dmat"))
+        _, BE, _, _, _, _ = igl.read_tgf(os.path.join(self.test_path, "arm.tgf"))
+
+        # Use same values as tutorial
+        # https://github.com/libigl/libigl/blob/main/tutorial/408_DirectDeltaMush/main.cpp
+        p = 20
+        l = 3
+        k = 1
+        a = 0.8
+        omega = igl.direct_delta_mush_precomputation(V, F,W, p, l, k, a)
+
+        self.assertTrue(omega.shape[0] == V.shape[0])
+        self.assertTrue(omega.shape[1] == BE.shape[0] * 10)
+
+        T = np.zeros((BE.shape[0]*4, 3))
+        I = np.eye(3)
+        for i in range(0, T.shape[0], 4):
+            T[i:i+3, :] = I
+
+        U = igl.direct_delta_mush(V, T, omega)
+
+        self.assertTrue(U.shape[0] == V.shape[0])
+        self.assertTrue(U.shape[1] == 3)
+        self.assertTrue(U.dtype == V.dtype)
+        self.assertFalse(np.isnan(U).any())
```

Comment:
add checks for dtypes

Reply:
I am assuming you meant dtype checks for omega and specific check for U. But if you have something else in mind please let me know